types/sanctuary/index.d.ts

```diff
@@ -107,19 +103,26 @@ declare namespace Sanctuary {
     gte<A>(x: Ord<A>): (y: Ord<A>) => boolean;
     min<A>(x: Ord<A>): (y: Ord<A>) => A;
     max<A>(x: Ord<A>): (y: Ord<A>) => A;
+    clamp<A = string | number | Date>(a: A): (b: A) => (c: A) => A;
```

Comment:
This union is not exhaustive.

<https://github.com/sanctuary-js/sanctuary-type-classes#lte>:

> `fantasy-land/lte` implementations are provided for the following built-in types: Null, Undefined, Boolean, Number, Date, String, Array, Arguments, and Object.

Ideally we would support all of these (in additional to the general `Ord a => a -> a -> a -> a`):

  - `clamp :: Null -> Null -> Null -> Null`
  - `clamp :: Undefined -> Undefined -> Undefined -> Undefined`
  - `clamp :: Boolean -> Boolean -> Boolean -> Boolean`
  - `clamp :: Number -> Number -> Number -> Number`
  - `clamp :: Date -> Date -> Date -> Date`
  - `clamp :: String -> String -> String -> String`
  - `clamp :: Ord a => Array a -> Array a -> Array a -> Array a`
  - `clamp :: Arguments -> Arguments -> Arguments -> Arguments` (questionable and unimportant)
  - `clamp :: Ord a => StrMap a -> StrMap a -> StrMap a -> StrMap a`